### ISSUE TITLE
Expanded admin/researcher search tools in "Manage Studies"

### ISSUE THREAD

**Issue by msheskin** [lookit/lookit-api#1254]
# TL;DR
Right now my option are to "filter name or description" in the search bar under "Manage Studies" and I want to be able to search by many more things (especially university name, PI name, full list of researchers, and UUID)

# Narrative
I often struggle to find studies that researchers refer to without the study name. For example, a researcher will contact me in Slack and say "My study doesn't have any participants. Is something wrong?" and I have to ask "What is the study name" to look it up (to tell them "you didn't mark it as discoverable"). As another example, researchers might submit a study and say "This is similar to the other study my lab posted, just with a different age range" and again I have no idea what other study to look up.

**Comment by becky-gilbert** [comment]
To add to this issue: on the Manage Study page, I would also like to be able to search studies by lab. My use case for this is that researchers could view all studies that are part of the "demo" lab, which would allow them to find demo studies more easily.

So here's the full list of things we'd like to be able to search for on the Manage Studies page:
- study name (already in place)
- description (already in place)
- name of researcher with any kind of access
- university name
- PI name
- UUID
- lab name

**Comment by becky-gilbert** [comment]
Another note about this issue. We are considering making changes to the "Responses" table on the Individual Responses page (#1497), and one proposed change is to improve the table's search/filter/sort functionality. If we do make that change, we could consider bumping this issue up since the changes might be very similar.
